lib/to_do/api.ex

```diff
@@ -0,0 +1,13 @@
+defmodule ToDo.API do
+  @file_path Application.compile_env(:http_server, :file_path, "lib/to_do/data/to_dos.json")
```

Comment:
does the `:file_path` belong in the `config`? Just like what you did with the integration tests config?
Said another way, should there also be a ToDoApi config?

Reply:
This seems like the kind of configuration you only want in 1 place in the code.

Reply:
I agree that I want a ToDoAPI config, but I also want to make sure the ToDoAPI can be used during the integration tests as well. I added `:file_path` to make sure my tests weren't overwriting data. However I'm realizing that probably means I should add a separate data file for the integration tests as well!